pkg/workloads/cortex/lib/api_utils.py

```diff
@@ -21,6 +21,9 @@
 from cortex.lib.log import cx_logger
 
 
+API_INFO_MESSAGE = "to make a prediction request, call this endpoint using the POST method and pass in the sample via JSON body"
```

Comment:
nit: send a POST request  to this endpoint with a sample in JSON to make a prediction